packages/react-scripts/template/README.md

```diff
@@ -584,7 +584,7 @@ To enable importing files without using relative paths, you can add the  `--incl
 
 ```
 "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
-"watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
+"watch-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
```

Comment:
the ` --recursive` flag was removed in `1.0.0` and recursion is always on because there is no way to be non-recursive with chokidar.

Other than that this LGTM

Reply:
also the `--include-path ./src` isn't necessary, let's say you have files `./src/index.scss` and  `./src/_mixins.scss` and in index.scss you `@import './_mixins'` it will still import correctly.

You don't necessarily need the `--include-path ./src`

Thanks @originell for keeping the docs updated with node-sass-chokidar, it had slipped my mind.